common/network-common/src/test/java/org/apache/spark/network/util/TransportFrameDecoderSuite.java

```diff
@@ -153,7 +148,7 @@ public void testEmptyFrame() throws Exception {
   @Test(expected = IllegalArgumentException.class)
   public void testLargeFrame() throws Exception {
     // Frame length includes the frame size field, so need to add a few more bytes.
-    testInvalidFrame(Integer.MAX_VALUE + 9);
+    testInvalidFrame((long) Integer.MAX_VALUE + 9);
```

Comment:
It looks like this call isn't invalid per se at the moment. There is a check later on in the code that compares a frame size to Integer.MAX_VALUE (but, as a long, so it works) and throws an exception, but that doesn't occur until later.